Sources/Layout/MessagesCollectionViewFlowLayout.swift

```diff
@@ -45,31 +44,51 @@ open class MessagesCollectionViewFlowLayout: UICollectionViewFlowLayout {
     open var messageLabelFont: UIFont {
```

Comment:
It may be nice to move the `font` into the `TextMessageLayout` object.
Pros:
- Then attributed text would have a default font (see #559 and #563)
-  More intuitive API
Cons:
- Then users are required to set an emoji font, we lose the current 2x by default behavior